Diagnostics/Get-NetView.PS1

```diff
@@ -1712,17 +1712,18 @@ function Worker {
     )
 
     $columns   = 4096
-    $version   = "17.06.1"
+    $version   = "17.06.2"
     $timestamp = Get-Date -f yyyy.MM.dd_hh.mm.ss
 
     # Input Validation
     CheckAdminPrivileges $SkipAdminCheck
-    $workDir   = (GetWorkDir $OutputDirectory)
-
-    Clear-Host
+    $workDir   = (NormalizeWorkDir $OutputDirectory)
 
     EnvSetup          -OutDir $workDir
 
+    # Start Run
+    Clear-Host
```

Comment:
Realized the `EnvSetup` can throw an exception based on user input. So the terminal should be cleared after `EnvSetup` so user can see input and exception together.